index.html

```diff
@@ -186,137 +202,129 @@ <h2>Multi-Post Stories</h2>
             </li>
           </ul>
-          <li class="dropdown">
-            <span class="dropdown-text">Frameworks</span>
-            <i class="fa fa-chevron-right"></i>
-          </li>
-          <li class="dropdown">
-            <span class="dropdown-text">Skills</span>
-            <i class="fa fa-chevron-right"></i>
-          </li>
-        </ul>
+        </div>
       </div>
     </section>
 
     <!-- CONTACT SECTION -->
-    <section class="contact-section">
-      <ul class="contact-content">
-        <li><h2 class="contact-title">Contact me</h2></li>
-        <li>
-          <p class="contact-text">
-            If you have an application you are interested in developing, a
-            feature that you need built or a project that needs coding. I’d love
-            to help with it
-          </p>
-        </li>
-
-        <li>
-          <form action="https://formspree.io/f/xvoyekjg" method="post">
-            <ul class="contact-form">
-              <li>
-                <label>
-                  <input
-                    class="input"
-                    type="text"
-                    id="name"
-                    name="name"
-                    placeholder="Enter your name"
-                    maxlength="30"
-                    required
-                  />
-                </label>
-              </li>
-              <li>
-                <label>
-                  <input
-                    class="input"
-                    type="email"
-                    id="email"
-                    name="email"
-                    placeholder="Enter your email"
-                    required
-                  />
-                </label>
-              </li>
-              <li>
-                <label>
-                  <textarea
-                    class="input"
-                    name="message"
-                    id="message"
-                    maxlength="500"
-                    placeholder="Write your message here"
-                    required
-                  ></textarea>
-                </label>
-              </li>
-              <li>
-                <button class="btn-contact" type="submit">Get in touch</button>
-              </li>
-            </ul>
-          </form>
-        </li>
-      </ul>
+    <section class="contact-section" id="contact">
+      <div class="contact-container">
+        <h2 class="contact-title">Contact me</h2>
+        <p class="contact-text">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+        <form action="https://formspree.io/f/xvoyekjg" method="post">
+          <label>
+            <input
+              class="input"
+              type="text"
+              id="name"
+              name="name"
+              placeholder="Enter your name"
+              maxlength="30"
```

Comment:
- [ ] Good job in designing your portfolio website :100: , but you should stick as much as possible to the Figma design. Kindly look :eyes:  at the image below and noticed that it could be better if you improve your design according to the Figma Template. :100: 

Your Design 

![Screenshot from 2022-12-08 19-02-58](https://user-images.githubusercontent.com/80458199/206520085-96ca7c1b-41d6-4f54-9d47-ad867934ef03.png)

Figma design

![Screenshot from 2022-12-08 19-09-55](https://user-images.githubusercontent.com/80458199/206520110-74dc4800-f40b-4319-91e2-4463d63ecf03.png)